### PR TITLE
✨ fix a v1 promotion doc mistake

### PR DESCRIPTION
**Pull request by grokspawn**
@LalatenduMohanty , @everettraven y'all were going so 🚀 on #1441 that I didn't get a chance to complete review.  It looks like you caught the things that I caught as well, except for this minor docs issue.  We could conceivably ignore this and it will become correct once we complete v1 api promotion in operator-controller.  

On the off case that we want to keep it consistent with context, here's this PR. 

# Description

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [ ] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

**Comment by everettraven** [comment]
Ah, thanks for catching this. We can merge this to main and it should be appropriately reflected in the docs site even after the release candidate release is finished

File: docs/howto/how-to-version-range-upgrades.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Set the version for the desired package in the Catalog source to a comparison st
 Example:
 
 ```yaml
-apiVersion: olm.operatorframework.io/v1
+apiVersion: olm.operatorframework.io/v1alpha1
```
Comment (everettraven) [inline docs/howto/how-to-version-range-upgrades.md:8]: This actually should be v1, not v1alpha1. We completed the v1 API bump for the ClusterExtension API as of #1228

Comment (grokspawn) [reply docs/howto/how-to-version-range-upgrades.md:8]: Right.  I was going to push changes to that PR, but it ... merged under me, and I mistakenly transplanted the v1alpha1 stuff.

**Review by m1kola — APPROVED** [review]

**Comment by m1kola** [comment]
Need https://github.com/operator-framework/operator-controller/pull/1451 to go in first, it seems. Or a force merge since this is just docs (I don't have permissions).

**Pull request closed without merging**